src/llmq/quorums_dkgsession.cpp

```diff
@@ -437,9 +438,47 @@ void CDKGSession::VerifyAndComplain(CDKGPendingMessages& pendingMessages)
     logger.Batch("verified contributions. time=%d", t1.count());
     logger.Flush();
 
+    if (sporkManager.IsSporkActive(SPORK_21_QUORUM_ALL_CONNECTED)) {
+        VerifyConnectionAndMinProtoVersions();
+    }
+
     SendComplaint(pendingMessages);
 }
 
+void CDKGSession::VerifyConnectionAndMinProtoVersions()
```

Comment:
Shouldn't we run this only for PoSe quorums? i.e introduce `consensus.llmqTypePoSe`, set it to `Consensus::LLMQ_400_85` for mainnet etc. and then `if (consensus.llmqTypePoSe != params.type) return;` here. (btw, is it me or is `dkgBadVotesThreshold` too low (300) for `LLMQ_400_85`?)

Reply:
The idea was to let all DKGs do these checks. And about LLMQ_400_85...I today believe that it was a mistake to include that one and would prefer to completely remove it. My plan was to create a PR in the next days that adds a BIP9 deployment to remove it forever and at the same time disable DKGs for it on spork21 activation.

And yes, `dkgBadVotesThreshold` is too low, but I'm not sure if this would cause real issues. If we disable this LLMQ type, we won't have issues.